[Bug]: Test cases fail reporting "'GrpcHandler' object has no attribute 'wait_for_loading_partitions_complete'"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: latest 
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2): 2.0.0rc10.dev6
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Failed cases:
```
[2022-01-16T16:29:17.430Z] FAILED testcases/test_connection.py::TestConnectionOperation::test_connection_connect_params[test_alias_nme]

[2022-01-16T16:29:17.430Z] FAILED testcases/test_utility.py::TestUtilityParams::test_wait_for_loading_collection_not_existed

[2022-01-16T16:29:17.430Z] FAILED testcases/test_utility.py::TestUtilityBase::test_wait_loading_collection_empty

[2022-01-16T16:29:17.430Z] FAILED testcases/test_utility.py::TestUtilityParams::test_wait_for_loading_partition_not_existed
```

```
[2022-01-16T16:29:17.430Z] [2022-01-16 16:19:36 - DEBUG - ci_test]: (api_request)  : [wait_for_loading_complete] args: ['coll__QZ4ZjNsA', ['1970_01_01'], 20, 'default'], kwargs: {} (api_request.py:55)
[2022-01-16T16:29:17.430Z] [2022-01-16 16:19:36 - ERROR - ci_test]: Traceback (most recent call last):
[2022-01-16T16:29:17.430Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 22, in inner_wrapper
[2022-01-16T16:29:17.430Z]     res = func(*args, **kwargs)
[2022-01-16T16:29:17.430Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 56, in api_request
[2022-01-16T16:29:17.430Z]     return func(*arg, **kwargs)
[2022-01-16T16:29:17.430Z]   File "/usr/local/lib/python3.6/site-packages/pymilvus/orm/utility.py", line 234, in wait_for_loading_complete
[2022-01-16T16:29:17.430Z]     return _get_connection(using).wait_for_loading_partitions_complete(collection_name,
[2022-01-16T16:29:17.430Z] AttributeError: 'GrpcHandler' object has no attribute 'wait_for_loading_partitions_complete'
```

### Expected Behavior

No exception

### Steps To Reproduce

```markdown
Running the following cases:

testcases/test_connection.py::TestConnectionOperation::test_connection_connect_params[test_alias_nme]

testcases/test_utility.py::TestUtilityParams::test_wait_for_loading_collection_not_existed

testcases/test_utility.py::TestUtilityBase::test_wait_loading_collection_empty

testcases/test_utility.py::TestUtilityParams::test_wait_for_loading_partition_not_existed
```


### Anything else?

_No response_

--- a/tests/python_client/requirements.txt
+++ b/tests/python_client/requirements.txt
@@ -12,7 +12,7 @@ allure-pytest==2.7.0
 pytest-print==0.2.1
 pytest-level==0.1.1
 pytest-xdist==2.2.1
-pymilvus==2.0.0rc10.dev7
+pymilvus==2.0.0rc10.dev11
 pytest-rerunfailures==9.1.1
 git+https://github.com/Projectplace/pytest-tags
 ndg-httpsclient

